low-level/files/opt/seagate/sspl/bin/conf_based_sensors_enable

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -e -u -o pipefail
+
+to_lower_case() {
+    tr '[:upper:]' '[:lower:]'
+}
+
+declare -a sed_expr=(-e '/^\[[A-Z]\+\]$/h')
+storage_type="$(provisioner get_setup_info --output json | jq -r .ret.storage_type | to_lower_case)"
+server_type="$(provisioner get_setup_info --output json | jq -r .ret.server_type | to_lower_case)"
+
+declare -A monitor=(
+    [REALSTORSENSORS]="true"
+    [NODEHWSENSOR]="true"
+    [SYSTEMDWATCHDOG]="true" # This is the 'server disks' sensor
+    [RAIDSENSOR]="true"
+    [SASPORTSENSOR]="true"
+    [MEMFAULTSENSOR]="true"
+    [CPUFAULTSENSOR]="true"
+
+)
+
+case "${storage_type}" in
+    virtual) 
+        monitor[REALSTORSENSORS]="false"
+        monitor[SYSTEMDWATCHDOG]="false"
```

Comment:
why systemdwatchdog sensor, meant for server local disk monitoring, disabled for external virtual storage type?

Reply:
Fixed as discussed here: https://teams.microsoft.com/_#/conversations/19:f13ac291ae4944db814d353d9c3f0b97@thread.v2?ctx=chat